_sass/blocks/jekyll-layouts/_state-pages.scss

```diff
@@ -53,11 +53,11 @@
   }
 
   p {
-    @include heading('para-md');
+    @include heading('para-lg');
```

Comment:
Since this is the same as the rest of the site, could we nix it for less confusion?